spec/index.html

```diff
@@ -375,6 +386,9 @@ <h3>Terms defined by this specification</h3>
           and SHOULD support the ability to specify other hash algorithms.
           Using a different hash algorithm will generally result in different output than
           using the default.</p>
+
+        <p class="note">There is no expectation that the default hash function
+          is also used to create a hash signature of the canonical N-Quads result.</p>
```

Comment:
For consistency ("function" => "algorithm") and minor wording adjustment:

```suggestion
        <p class="note">There is no expectation that the default hash algorithm
          will also be used by any application creating a hash digest of the
          canonical N-Quads result.</p>
```